packages/solicitudes/src/components/ModuleSelector/NuevaSolicitud/index.jsx

```diff
@@ -9,18 +9,18 @@ function NewRequest() {
       <Grid container spacing={2}>
         <Grid item xs={5}>
           <Input
-            label="Primer Apellido"
-            id="lastname1"
-            name="lastname1"
-            auto="lastname1"
+            label="Modalidad"
+            id="modalidad"
+            name="modalidad"
+            auto="modalidad"
```

Comment:
I'd prefer this one was an compponent

Reply:
It is a component :) it just requires all of those props

Reply:
Okay